### DISCUSSION TITLE
CSV Export #2

### DISCUSSION THREAD

**Discussion by guntherweissenbaeck** [guntherweissenbaeck/django_fbf#44] [Announcements]
- Bitte den letzten Commit pullen und in den issue2 branch auschecken. 
- im Adminbereich unter Gruppen "data-export" anlegen
- unter Benutzer den Benutzer auswählen der CSV-Export-berechtigt sein soll
- unter Benutzer->Berechtigungen->Gruppen->verfügbare Gruppen den Eintrag "data-export" nach rechts verschieben
- sichern der Benutzereinstellungen
- als soeben modifizierten Benutzer anmelden und prüfen, ob in der Navigationsliste der Eintrag "Data-Export" erscheint.
- Bitte um Amerkungen, Vorschläge und Kritik

**Comment by Java-Fish** [comment]
Kann leider in der Navigationsliste den Eintrag Data-Export nicht finden.

**Comment by Java-Fish** [comment]
Export funktioniert einwandfrei, erst einmal keine Änderungswünsche

**Comment by guntherweissenbaeck** [comment]
Deployed
